lib/components/Notification/notification-icon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    StandaloneCircleCheckBoldIcon,
+    StandaloneCircleExclamationBoldIcon,
+    StandaloneCircleInfoBoldIcon,
+    StandaloneTriangleExclamationBoldIcon,
+} from "@deriv/quill-icons";
+import { TYPE } from "../../utils/notification-utils";
+
+interface NotificationIconProps {
+    type?: (typeof TYPE)[keyof typeof TYPE];
```

Comment:
Why is this optional?

Reply:
I think the majority of the notifications are `info` and thus we have it as default type already.

Reply:
yes, that's right )